lib/src/layer/tile_layer.dart

```diff
@@ -327,16 +325,18 @@ class _TileLayerState extends State<TileLayer> with TickerProviderStateMixin {
         ),
       )..listen(_update);
     }
+
+    super.initState();
```

Comment:
Why is `super.initState` called last? Only `super.dispose` is advised to be called last.

Reply:
I'm following instruction from Dart/Flutter Plugin for VSCode only

```@override
  void initState() {
    // TODO: implement initState
    super.initState();
  }
```

Reply:
I’lol update this line to first line :)